documentation/api/trace-custom.md

```diff
@@ -24,6 +24,8 @@ The default host address for these routes is `https://localhost:52323`. This rou
 | `durationSeconds` | query | false | int | The duration of the trace operation in seconds. Default is `30`. Min is `-1` (indefinite duration). Max is `2147483647`. |
 | `egressProvider` | query | false | string | If specified, uses the named egress provider for egressing the collected trace. When not specified, the trace is written to the HTTP response stream. See [Egress Providers](../egress.md) for more details. |
 
+> **NOTE:** When setting `durationSeconds` to `-1` (indefinite duration), there is currently no way to terminate the trace operation that preserves the `.nettrace` file in an accessible format. This also applies when prematurely terminating a trace operation that uses a finite value for `durationSeconds`.
```

Comment:
I would suggest moving this to the Additional Notes section since it is not pertinent to the parameters.